src/components/utils.js

```diff
@@ -18,13 +18,15 @@ export function cellLayerDefaultProps(cells, updateStatus, updateCellsHover, uui
     getElevation: 0,
     getLineWidth: 0,
     onHover: (info) => {
-      if (info.object) {
-        updateStatus(makeCellStatusMessage(info.object[1].factors));
+      if (info.object && Array.isArray(info.object) && info.object.length === 2) {
```

Comment:
This feels more like a check where, if it wasn't an array, something was really wrong, and we'd want to know about it.

Reply:
That makes sense, I have removed the check